front/app/containers/Admin/dashboard/components/ChartFilters/messages.ts

```diff
@@ -0,0 +1,28 @@
+import { defineMessages } from 'react-intl';
```

Comment:
These used to be in `front/app/containers/Admin/dashboard/messages.ts`, just moved them closer to where they are being used.